pyinterface/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.14...3.31)
+
+set(UseSWIG_MODULE_VERSION 2)
+include(UseSWIG)
+
+# Really should be 3.17+ and Development.Module, otherwise you require unnecessary libs
+find_package(Python COMPONENTS Interpreter Development)
+
+set_property(SOURCE fastjet_swig.i PROPERTY CPLUSPLUS ON)
+
+swig_add_library(fastjet_swig LANGUAGE python SOURCES fastjet_swig.i OUTPUT_DIR fastjet_swig)
+set_property(TARGET fastjet_swig PROPERTY SWIG_USE_TARGET_INCLUDE_DIRECTORIES TRUE)
+# Unix only, Windows is different
+set_property(TARGET fastjet_swig PROPERTY SUFFIX ".${Python_SOABI}${CMAKE_SHARED_MODULE_SUFFIX}")
+
+target_link_libraries(fastjet_swig PUBLIC fastjet)
+target_link_libraries(fastjet_swig PUBLIC fastjettools)
+target_link_libraries(fastjet_swig PUBLIC Python::Module)
+
+target_include_directories(fastjet_swig PUBLIC ../pyinterface)
+
+# Really should go to ${Python_SITEARCH} or into the wheel dir of scikit-build-core
+file(RENAME "${CMAKE_CURRENT_BINARY_DIR}/fastjet_swig/fastjet_swig.py" "${CMAKE_CURRENT_BINARY_DIR}/fastjet_swig/__init__.py")
```

Comment:
Running (using the above `pixi.toml` (I've edited it to force the `start` task to clone the repo fresh)

```
pixi clean && pixi run start
```

in a new directory hits

```
CMake Error at pyinterface/CMakeLists.txt:23 (file):
  file RENAME failed to rename

    /tmp/debug-fastjet-cmake/build/pyinterface/fastjet_swig/fastjet_swig.py

  to

    /tmp/debug-fastjet-cmake/build/pyinterface/fastjet_swig/__init__.py

  because: No such file or directory



-- Configuring incomplete, errors occurred!
```

Reply:
oh that's weird - let me mess around with it.

Reply:
See https://cmake.org/cmake/help/latest/module/UseSWIG.html, use SWIG_MODULE_NAME.

Reply:
ok! latest commit should do it correctly now